akka-docs/rst/scala/http/routing-dsl/directives/basic-directives/extractStrictEntity.rst

```diff
@@ -0,0 +1,28 @@
+.. _-extractStrictEntity-:
+
+extractStrictEntity
+===================
+
+Signature
+---------
+
+.. includecode2:: /../../akka-http/src/main/scala/akka/http/scaladsl/server/directives/BasicDirectives.scala
+   :snippet: extractStrictEntity
+
+Description
+-----------
+
+Extracts the strict http entity as ``HttpEntity.Strict`` from the :class:`RequestContext`.
+
+A timeout parameter is given and if the stream isn't completed after the timeout, the directive will be failed.
+
+.. warning::
+
+  The directive will read the entire request entity into memory regardless of size and effectively disable streaming.
```

Comment:
Does this really render into a warning box (when there is a separating linefeed)?